pkg/apis/config/v1alpha1/interpretercontext_types.go

```diff
@@ -62,7 +62,8 @@ type ResourceInterpreterRequest struct {
 	DesiredReplicas *int32 `json:"replicas,omitempty"`
 
 	// AggregatedStatus represents status list of the resource running in each member cluster.
-	// +optional
+        // It'll be set only if InterpreterOperation is InterpreterOperationAggregateStatus.
```

Comment:
Actually, it always is set, no matter [AggregateStatus](https://karmada.io/docs/userguide/globalview/customizing-resource-interpreter/#aggregatestatus) is enabled or not for the resource.

Reply:
Oh, sorry, my bad. You are right! Ignore me.

I thought the comments is added to here:
https://github.com/karmada-io/karmada/blob/4011f07178c97648ce547640f04d240dd38d8004/pkg/apis/work/v1alpha2/binding_types.go#L253-L255